tests/full-example.ds

```diff
@@ -12,17 +12,15 @@ struct vec3 {
 
 impl vec3 {
   fn length(): f32 {
-    f32(sqrt(self.x*self.x + self.y*self.y + self.z*self.z));
+    return f32(sqrt(self.x*self.x + self.y*self.y + self.z*self.z));
   }
   
   fn to_str(): str {
     format("Vec3 {X: %d, Y: %d, Z: %d}", self.x, self.y, self.z);
```

Comment:
implicit return statements don't have a semicolon at the end iirc

Reply:
Fixed

Reply:
you fixed the wrong one 🤦‍♂️

Reply:
What do you mean? I removed the semicolon after the implicit return. There was only one. The one you're by is the explicit return, and that should have a semicolon iirc.

Reply:
Ah. Fixing...